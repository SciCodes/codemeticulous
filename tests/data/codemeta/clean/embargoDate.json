{
  "@context": "https://w3id.org/codemeta/3.0",
  "@type": "SoftwareSourceCode",
  "name": "Embargoed Software",
  "embargoDate": "2023-12-31"
}
