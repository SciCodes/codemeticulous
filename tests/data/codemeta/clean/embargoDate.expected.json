{
  "@context": "https://w3id.org/codemeta/3.0",
  "@type": "SoftwareSourceCode",
  "name": "Embargoed Software",
  "embargoEndDate": "2023-12-31"
}
